impl/src/main/java/org/jboss/cdi/tck/tests/context/dependent/TransactionalInterceptor.java

```diff
@@ -17,14 +17,16 @@
 package org.jboss.cdi.tck.tests.context.dependent;
 
 import jakarta.annotation.PreDestroy;
+import jakarta.annotation.Priority;
 import jakarta.enterprise.context.Dependent;
 import jakarta.interceptor.AroundInvoke;
 import jakarta.interceptor.Interceptor;
 import jakarta.interceptor.InvocationContext;
 
-@Dependent
+//@Dependent
```

Comment:
This should left intact IMHO. I think the spec allows interceptor classes to be declared `@Dependent` (but not any other scope): "If an interceptor declares any scope other than @Dependent, the container automatically detects the problem and treats it as a definition error."

Reply:
Yeah, placing a `@Dependent` annotation on an interceptor makes no difference. Placing any other scope on it should result in an error,

Reply:
I left the annotation